src/test/emulators/functionsRuntimeWorker.spec.ts

```diff
@@ -99,16 +111,18 @@ describe("FunctionsRuntimeWorker", () => {
       const worker = new RuntimeWorker(workerPool.getKey("trigger"), new MockRuntimeInstance());
       const counter = new WorkerStateCounter(worker);
 
+      worker.readyForWork();
```

Comment:
(sorry for adding busy work) can we update the title to include the new `created` state?

Reply:
updated